ion/bitstream.go

```diff
@@ -626,12 +641,12 @@ func (b *bitstream) readNsecs(len uint64) (int, bool, error) {
 
 // ReadDecimal reads a decimal value of the given length: an exponent encoded as a
 // varInt, followed by an integer coefficient taking up the remaining bytes.
-func (b *bitstream) readDecimal(len uint64) (*Decimal, error) {
+func (b *bitstream) readDecimal(length uint64) (*Decimal, error) {
 	exp := int64(0)
 	coef := new(big.Int)
 
-	if len > 0 {
-		val, _, vlen, err := b.readVarIntLen(len)
+	if length > 0 {
+		val, _, vlen, err := b.readVarIntLen(length)
```

Comment:
Since we're changing `len` to `length`, how about we change `vlen` to `vlength`?

Reply:
Likewise change `olen` to `olength` in readTimestamp()?